ipynb/tests/Frequency_Invariance_Test.ipynb

```diff
@@ -0,0 +1,827 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f9dcb170750>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_signal_against_freq('util_avg')"
```

Comment:
As discussed, it would be best if the load/util_avg axes had a wider range as to show that you should have almost the same value for all frequencies.